js/mock_builtin.js

```diff
@@ -0,0 +1 @@
+export default undefined;
```

Comment:
Is this really needed?

Reply:
We cannot leave the imports of the Node.js builtins undefined.  We have to provide something.  I think I can do it just within the rollup config and then drop this.

Reply:
I tried, but there is no straightforward way because of a defect with rollup/rollup-plugin-commonjs#315, so for now, an empty module like this needs to be used... I put comments in `rollup.config.js` to that effect.  Once resolved, then `rollup-plugin-virtual` can be used and the mock just becomes part of `rollup.config.js`.